app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionActivity.java

```diff
@@ -55,6 +58,17 @@ protected void onCreate(Bundle savedInstanceState) {
         answerList.setLayoutManager(manager);
     }
 
+    private void setupAnswerMinAlert(Question question) {
+        TextView alert = findViewById(R.id.question_answers_alert);
+        state.getNbCheckedAnswer().observe(this, nbrVotes -> {
+            if (question.getAnswerMin() < nbrVotes ) {
+                alert.setText(R.string.answers_min_alerts);
+            } else {
+                alert.setText("");
+            }
+        });
```

Comment:
Rather than setting the text dynamically, I highly suggest to use `View#setVisiblity(...)` with the constants `HIDDEN` and `VISIBLE`. On top of that, if you add the `animateLayoutChanges` to the XML layout that contains the view, some default animations will be provided "for free".